examples/packages/single-language/haskell-cabal/README.md

```diff
@@ -0,0 +1,13 @@
+# Haskell Cabal build
+
+## Build project
+
+```console
+$ nix build .#checks.x86_64-linux.example-package-haskell-cabal
+```
+
+## Generate lock file
+
+```console
+$ nix run .#checks.x86_64-linux.example-package-haskell-cabal.lock
```

Comment:
I really appreciate the effort to add some docs, but these attribute paths are not portable.
Once the example gets copied to another flake, these attribute paths are not guaranteed to be the same.

Maybe remove this file and extend the `./docs` instead by adding a guide (can be separate PR)?